examples/constructor/src/builder.rs

```diff
@@ -297,20 +297,13 @@ impl Calls {
     pub fn if_else(
```

Comment:
some tests, especially returning some value from if_else would be great

Reply:
Test still actual. Impl it with return of some value from block. Use unit tests with non-related to syscalls commands